astropy/coordinates/tests/test_pickle.py

```diff
@@ -64,7 +64,7 @@ def test_pickle_longitude_wrap_angle():
 
 @pytest.mark.parametrize(("name", "args", "kwargs", "xfail"),
                          tuple(zip(_names, _args, _kwargs, _xfail)))
-def test_simple_object(pickle_protocol, name, args, kwargs, xfail):
+def test_simple_object(pickle_protocol, name, args, kwargs, xfail):  # noqa: F811
```

Comment:
It would be nice to ignore this noqa, but ONLY in tests. Pytest fixtures do not play nicely with PEP8.

Reply:
Looking at https://github.com/pre-commit/pygrep-hooks/blob/main/.pre-commit-hooks.yaml, I don't think refinement is possible, so the noqa is fine.